templates/kubernetes/config.yml

```diff
@@ -0,0 +1,5 @@
+name: Kubernetes
+description: |
+  Kubernetes service
+version: v0.1.0-rancher1
+category: Rancher System Services
```

Comment:
@vincent99 introduced "Rancher System Services" category so we can filter out k8s system template from the Catalog UI. Other naming suggestions are welcome.